PacketDotNet/PacketDotNet.csproj

```diff
@@ -14,6 +14,12 @@
     <Product />	
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
These changes seem unrelated to the rest of the commit and should go into another commit.